LICENSE.md

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2015, Sky UK Ltd All rights reserved.
```

Comment:
build fail because the file upstream was changed so different checksum related to get file to upstream to fix pagination getAll repo (#1) rebase the branch and update to 2017